pyshelf/bulk_update/utils.py

```diff
@@ -13,6 +14,40 @@ def run(args):
                 options provided to the update-search-index
                 script
     """
+    container = _configure(args, "update-search-index")
+    bucket_string = args.get("--bucket")
+    bucket_list = []  # heh
```

Comment:
I'm confused by this additonal.  I recognize it as something I wrote earlier, however, I don't see it being moved and I don't see it more than once in the code base.  Do you know what happened here?